src/components/02-components/card-experiences-list/card-experiences-list.tsx

```diff
@@ -0,0 +1,69 @@
+import { Box, Paper } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import CardExperiences from '../card-expreciences/card-experiences';
+
+
+export type BrandProps = {
```

Comment:
Why `Brand`? Use the correct name for the component. Also, don't export the type if you don't need to import it from another component.